geonetwork/gn-tool/gn3-tool.sh

```diff
@@ -590,6 +601,7 @@ main() {
             -p) gn_password="$2"; shift 2;;
             -y) uuid_action="$2"; shift 2;;
             -z) group="$2"; shift 2;;
+            -a) harvest_all="$2"; shift 2;;
```

Comment:
Geared up to take a parameter?  May be confusing down the track. The presence of the -a option is enough to signal you want to export harvested records I would have thought

Reply:
I was hoping you wouldn't notice. No iv'e got to understand this bash junk